### ISSUE TITLE
4gamer.net

### ISSUE THREAD

**Issue by otksm** [Yuki2718/adblock#30]
### URL(s) where the issue occurs (mandatory)

https://www.4gamer.net

### Describe the issue (mandatory)

`||4gamer.net/img/*_background_*jack_$image,1p`

Not sure if this was meant to be a mobile-only filter but _theoretically_ this does not cover cases such as `https://www.4gamer.net/img/enderlilies_background_jack.jpg` on desktop (see https://web.archive.org/web/20210624030248/https://www.4gamer.net/), although this is covered by the fallback cosmetic filter `4gamer.net##body:style(background:none!important)`.

Blocking the background also only hides parts of the background ad since some of it is embedded in the `.header` element. The following filters block the remaining part of the ad while maintaining some readability in the header. It's not an exact match but it is as close as I managed (for references see https://www.4gamer.net/nds OR https://web.archive.org/web/20210716054215/https://www.4gamer.net/). I have separated this issue from the previous one as you would probably consider this out of scope.

```
||4gamer.net/img/*_header_*jack$image,1p
!#if !env_mobile
4gamer.net##.V2_utility:style(background-image: linear-gradient(#747474, #585858 10%, #3c3c3c 90%, #525252 95%) !important; outline: 1px solid black !important; outline-offset: -1px !important)
4gamer.net##.header:style(background-image: linear-gradient(#ffffff, #f0f0f0) !important)
4gamer.net##dl.search:style(color: #333333 !important)
4gamer.net##dt.search_middle > a[href="/specials/search_middle/"]:style(color: #000080 !important)
!#endif
```

### Versions (mandatory)

- Browser and version: Firefox Desktop
- uBlock Origin version: 1.36.2

### Filters (mandatory)

Yuki's uBlock Japanese filters

**Comment by Yuki2718** [comment]
> Not sure if this was meant to be a mobile-only filter 

Yuki's uBlock Japanese filter is designed and optimized for PC only. The only reason it includes a few mobile rules is that I can't really force user to read README-JP thoroughly and not to use the list on mobile.

The header is intentionally unblocked as this is not as obtrusive as sidebar counterparts. I see this is now causing a bit ugly layout but it's still not a serious issue. I'm reluctant to use `:style()` filters excessively even tho in this case the performance cost can well be paid out by the image not being loaded.
